### ISSUE TITLE
Alacritty doesn't import settings from external file

### ISSUE THREAD

**Issue by aagl** [alacritty/alacritty#4269]
OS: Linux/wayland
Version:  alacritty 0.5.0

My config was getting rather big and i wanted to split it, but i can't get
alacritty to import settings from external file. It seems anything under 
'import' is just ignored. Also, it does not complain when i try to import from 
non-existing files or if i just put some nonsene under 'import':

Config:
```
import:
  - /home/a/.config/alacritty/colors.yml
  - /home/a/.config/alacritty/laptop.yml
  - /this/does/not/exist
  - this is just garbage
...
```

Log:
```
[2020-09-29 21:45:21.604439173] [INFO] Welcome to Alacritty
[2020-09-29 21:45:21.604493453] [INFO] Configuration loaded from "/home/a/.config/alacritty/alacritty.yml"
[2020-09-29 21:45:21.614702367] [DEBUG] Estimated DPR: 2
[2020-09-29 21:45:21.614723146] [DEBUG] Estimated Cell Size: 15 x 37
[2020-09-29 21:45:21.614731155] [DEBUG] Estimated Dimensions: None
[2020-09-29 21:45:21.698672423] [INFO] Device pixel ratio: 1
[2020-09-29 21:45:21.703276903] [INFO] Initializing glyph cache...
[2020-09-29 21:45:21.724409275] [INFO] ... finished initializing glyph cache in 0.021093607s
[2020-09-29 21:45:21.724441656] [INFO] Cell Size: 7 x 18
[2020-09-29 21:45:21.724447027] [INFO] Padding: 1 x 3
[2020-09-29 21:45:21.724454324] [INFO] Width: 800, Height: 600
[2020-09-29 21:45:21.725593565] [INFO] PTY dimensions: Line(33) x Column(114)
[2020-09-29 21:45:21.727233476] [INFO] Initialisation complete
[2020-09-29 21:45:21.731353425] [DEBUG] New num_cols is 178 and num_lines is 37
[2020-09-29 21:45:21.732783303] [INFO] Width: 1252, Height: 675
[2020-09-29 21:45:21.847222271] [INFO] Font size changed to Size(22) with DPR of 2
[2020-09-29 21:45:21.851557373] [DEBUG] New num_cols is 166 and num_lines is 36
[2020-09-29 21:45:21.851600313] [INFO] Width: 2504, Height: 1350

```

**Comment by chrisduerr** [comment]
>  Version: alacritty 0.5.0

It's not supported in 0.5.0. This is a feature that will be available in 0.6.0.

**Comment by aagl** [comment]
yup. works great with 0.6.0, thank you.
